### ISSUE TITLE
Deprecated call to install_in_connector()

### ISSUE THREAD

**Issue by thomich** [OCA/connector-magento#100]
File/Line: https://github.com/OCA/connector-magento/blob/8.0/magentoerpconnect/connector.py#L27

WARNING dbname openerp.addons.magentoerpconnect.connector: Deprecated: <module> at line 27: This call to 'install_in_connector()' has no effect and is not required.

Commented it locally and works fine....
Haven't made a PR, cause it's a smart issue ;)

**Comment by guewen** [comment]
Yes `install_in_connector()` is no longer required (yeah!) and we can remove it. Not a high priority issue but an easy pick. (should be removed from the doc as well)
